<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchoolDistributionAnalyzerDialogBase</class>
 <widget class="QDialog" name="SchoolDistributionAnalyzerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EduAccess Mapper</string>
  </property>
  <widget class="QLabel" name="labelSchool">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select School Layer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSchool">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="labelCities">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Cities Layer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCities">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Analysis</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
